[DOCU-2314] Konnect Service Hub: UI validation and style cleanup (#4082)
Kong/docs.konghq.com: app/konnect/servicehub/index.md

## Services in the Service Hub catalog

Each entry in the Service Hub is called a _{{site.konnect_short_name}} service_, or _service_

> for short. This is the abstraction of one of your own upstream services.

Author: Adjusted your comment for typo + shortname.

> **Note:** Currently, the only supported implementation type is a
{{site.base_gateway}} runtime.

The main attribute of a service version is its upstream URL, where the service
listens for requests. You can specify the URL with a single string, or by
specifying its protocol, host, port, and path individually.

[Get started with service management &rarr;](/konnect/servicehub/manage-services)

### {{site.base_gateway}} implementations

When configuring a {{site.base_gateway}} implementation of a service, you'll

> need to specify a route. Routes determine how (and if) requests get sent to

Reviewer: ```suggestion
need to specify a route. Routes determine how successful requests are sent to
```
